django/contrib/admin/tests.py

```diff
@@ -97,14 +99,29 @@ def wait_until_invisible(self, css_selector, timeout=10):
             timeout
         )
 
-    def wait_page_loaded(self):
+    def wait_page_ready(self, timeout=10):
```

Comment:
Make `timeout` keyword-only?
```python
def wait_page_ready(self, *, timeout=10):
    ...
```
Was there any educated choice behind the choice of 10 seconds? Would 5 or event 3 be enough for most test cases?

Reply:
10 seconds is standard in all other methods on this class.

I don't want to make it a keyword only options, as it would be inconsistent. If we change it, we should do it for the whole class and in a separate ticket. That being said, this is not public API.